twig/BS3/field_parents/select2HTML.twig

```diff
@@ -1,5 +1,11 @@
 {% macro render(field, multi) %}
-    <input type="hidden" name="{{ field.name }}" id="{{ field.id }}" class="{{ field.class }}" {{ field.getDataAttributeString()|raw }}>
+    <input type="hidden"
+           name="{{ field.name }}"
+           id="{{ field.id }}"
+           class="{{ field.class }}"
+           {{ (field.getAutoComplete() = '' ? 'autocomplete="off"' : 'autocomplete="' ~ field.getAutoComplete() ~ '"') }}
```

Comment:
For every one of these that you're changing, make sure you're changing the accompanying PHP file in src/Fields to ensure the method exists.

You may want to add your autoComplete method to a Trait (in the traits folder), then USE it in FormField and in any other fields that will have the autoComplete method

Reply:
Added changes to PHP files.  Made trait and applied it to FormField and all php files where the associated twig file was changed!